jupyter_book/book_template/content/features/notebooks.ipynb

```diff
@@ -491,11 +491,69 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "source": [
+    "## Executing notebooks\n",
+    "\n",
+    "✨**experimental**✨\n",
+    "\n",
+    "By default, Jupyter Book will not run any of your notebooks when you build your\n",
+    "book. It assumes you have already run each one, and simply converts each page to\n",
+    "HTML.\n",
+    "\n",
+    "However, sometimes you'd like to store *just* the book content, and then *run* this content\n",
+    "when you build your book. This helps you ensure that your content still runs, and\n",
+    "that you've always got the latest outputs in your book. The drawback to this is that\n",
+    "your book will take longer to build, especially if your content takes a long time\n",
+    "to run."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note:** If you store your Notebook content as [Jupytext documents](jupytext.html), then\n",
+    "they will automatically be run each time you build your book. This is because Jupytext\n",
```

Comment:
This suggests a difference in the way `ipynb` docs are treated compared to `md`/`py` docs?

| Type | Default | `execute`|
|---|---|---|
| `.ipynb`| Doesn't execute| Executes |
| `.md`| Executes |  Executes|
| `.py`| Executes |Executes |

I was wondering whether it would make sense to have settings in the `config.py` to define whether different things execute.

This could then allow for switches on different file types - so for example, you might run `ipynb` and ``md` but not disable execution of `py`?

Another way might be to enable / disable execution via config file in different directories, but I see this could get to be a config minefield:-(

Reply:
I really like each of these suggestions - do you think you could copy/paste them into an issue so we don't lose track of it? I don't have time to implement changes now but wanna keep track of the improvement